tests/core/fbtests/fbtesterglobalfixture.h

```diff
@@ -14,7 +14,34 @@
 
 #include "device.h"
 
-class CTesterDevice;
+class CTesterDevice : public CDevice {
+  public:
+    CTesterDevice(const SFBInterfaceSpec *paInterfaceSpec, const CStringDictionary::TStringId paInstanceNameId) :
+        CDevice(paInterfaceSpec, paInstanceNameId){
+    }
+
+    void awaitShutdown() override {
+      // nothing to be done to join
+    }
+
+    CIEC_ANY *getDI(size_t) override {
+      return nullptr;
+    }
+
+    CDataConnection **getDIConUnchecked(TPortId) override {
+      return nullptr;
+    }
+
+};
+
+const static SFBInterfaceSpec gscTestDevSpec = {
```

Comment:
this could be a private const static field of CTesterDevice and directly be given to CDevice. This would reduce the interface of the CTesterDevice. Didn't notice this when I changed the code. But now that you make that publicly visible I think this should be repaired.

Reply:
right! done!